File: week1-python/Getting_started_with_images/Manipulating_Pixels.py
# Import libraries
import cv2
import numpy as np
from dataPath import DATA_PATH

imagePath = DATA_PATH + "/images/number_zero.jpg"

# Read image in Grayscale format
testImage = cv2.imread(imagePath,0)

#access the first element
print(testImage[0,0])

#Modifying pixel value
testImage[0,0]=200
print(testImage)

File: week1-cpp/Getting_started_with_images/Manipulating_Pixels.cpp
#include <iostream>
#include "dataPath.hpp"
#include <opencv2/opencv.hpp>
#include <opencv2/core.hpp>
#include <opencv2/highgui.hpp>

using namespace std;
using namespace cv;

int main(void)
{
        string imagePath = DATA_PATH + "/images/number_zero.jpg";

        // Read image in Grayscale format
        Mat testImage = imread(imagePath,0);

	cout << (int)testImage.at<uchar>(0,0);

	testImage.at<uchar>(0,0)=200;

	cout << testImage;
	
	return 0;
}
